lib/server.go

```diff
@@ -457,10 +467,9 @@ func NewServer(
 	timesource := chainlib.NewMedianTime()
 
 	// Create a new connection manager but note that it won't be initialized until Start().
-	_incomingMessages := make(chan *ServerMessage, (_targetOutboundPeers+_maxInboundPeers)*3)
+	_incomingMessages := make(chan *ServerMessage, 100+(_targetOutboundPeers+_maxInboundPeers)*3)
```

Comment:
Why is this a hardcoded value of 100? I assume you want to use the number of validators from GlobalParams?